lms/djangoapps/commerce/views.py

```diff
@@ -65,6 +67,10 @@ def checkout_receipt(request):
             "If your course does not appear on your dashboard, contact {payment_support_link}."
         ).format(payment_support_link=payment_support_link)
 
+    if is_payment_complete:
+        cache_key = CommerceConfiguration.CACHE_KEY + '.' + request.user.username
```

Comment:
before deleting or performing an operation on commerce cache check if its enabled or not by setting a property method on commerce config model.
```python
    @property
    def is_cache_enabled(self):
        """Whether responses from the Commerce API will be cached."""
        return self.cache_ttl > 0
```